index.js

```diff
@@ -31,143 +31,124 @@ if(!heimdall.hasMonitor('babel')) {
   });
 }
 
-module.exports = Babel;
-function Babel(inputTree, _options) {
-  if (!(this instanceof Babel)) {
-    return new Babel(inputTree, _options);
-  }
+module.exports = class Babel extends Filter {
+  constructor(inputTree, options = {}) {
+    options.persist = 'persist' in options ? options.persist : true;
+    options.async = true;
 
-  let options = _options || {};
-  options.persist = 'persist' in options ? options.persist : true;
-  options.async = true;
+    super(inputTree, options);
 
-  if (options.browserPolyfill) {
```

Comment:
Need to update the README.md for this option's removal, though if you want to bring it back and have a PR dedicated to dropping the browser polyfill that might be better (for changelog/review/tracking purposes).

Reply:
Done, opened #191